packages/tc-api-db-manager/index.js

```diff
@@ -27,20 +28,39 @@ const initConstraints = async () => {
 			);
 		};
 
+		const retrieveIndexes = async () => {
+			const indexes = await executeQuery('CALL db.indexes');
+			return indexes.records.map(constraint =>
+				constraint.get('description'),
+			);
+		};
+
 		const existingConstraints = await retrieveConstraints();
-		const desiredConstraints = []
+		const existingIndexes = await retrieveIndexes();
+		const existingConstraintsAndIndexes = existingConstraints.concat(
+			existingIndexes,
+		);
+
+		const desiredConstraintsAndIndexes = []
 			.concat(
 				...schema.getTypes().map(({ name: typeName, properties }) => {
 					return [].concat(
 						...Object.entries(properties).map(
-							([propName, { unique }]) => {
-								return [
-									unique &&
+							([propName, { unique, canIdentify }]) => {
+								if (unique) {
+									return [
 										`CONSTRAINT ON (s:${typeName}) ASSERT s.${propName} IS UNIQUE`,
+									];
 									// skip the setting of enterprise version specific constraints until we use the enterprise version
-									// required &&
-									// 	`CONSTRAINT ON (s:${typeName}) ASSERT exists(s.${propName})`
-								];
+									// 	// required &&
+									// 	// 	`CONSTRAINT ON (s:${typeName}) ASSERT exists(s.${propName})`
+									// ];
+								}
+								if (canIdentify) {
+									return [
+										`INDEX ON :${typeName}(${propName})`,
+									];
+								}
```

Comment:
does it need to `return []` by default in order to avoid `desiredConstraintsAndIndexes` containing `undefined` items?